backend/src/akvo/lumen/component/keycloak.clj

```diff
@@ -93,6 +110,32 @@
         (response {:status (:status ed)
                    :body (:reasonPhrase ed)})))))
 
+(defn tenant-admin?
+  [request-draft api-root tenant user-id]
```

Comment:
@kardan could you explain what this `request-draft` object is? I see it's being used a lot but I'm not sure I understand what it is